llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -64,6 +65,50 @@ cl::opt<RelEncodeLoc> RelocLocation(
                clEnumVal(CompileUnit, "In compile unit")),
     cl::init(Atom));
 
+static bool isScatteredRelocation(const MachO::any_relocation_info &RelocInfo,
```

Comment:
High level question about all these functions doing bit manipulation: are there any LLVM utilities we can re-use here?